pandexo/engine/run_online.py

```diff
@@ -502,21 +499,19 @@ def post(self):
         or `self.request.body` to grab the entire returned object.
         """
         
-        #print(self.request.body)
-        
-        id = str(uuid.uuid4())+'h'
+        # print(self.request.body)
 
+        id = str(uuid.uuid4())+'h'
         with open(os.path.join(os.path.dirname(__file__), "reference",
                                "exo_input.json")) as data_file:
             exodata = json.load(data_file)
             exodata["telescope"] = 'hst'
 
-
             #planet properties 
             properties = self.get_argument("properties")
-            
-            if properties=="user":
 
+            if properties=="user":
+                print('bad')
```

Comment:
Is this print statement supposed to be here?

Reply:
Sorry, was using that for testing.